CreateAnExperiment.md

```diff
@@ -14,4 +14,6 @@ Experiments should be contained within a branch in the dotnet/runtimelab reposit
    - Update the pre-release label to include a unique identifier representing the name of the experiment to avoid package clashes given that all experiments publish to the same [feed](https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-experimental). To do this you need to update the versioning properties in [`Versions.props`](https://github.com/dotnet/runtimelab/blob/0cf87055346fd12fb22478f17521ebeb28a6d323/eng/Versions.props#L9). Make sure the label you choose is maximum 7 chars long as NuGet has a limit on the version length so the official build would fail. 
    - Edit `eng/pipelines/runtimelab.yml` in your branch to just build what your experiment needs on CI.
    - To avoid spurious github notifications for merges from upstream, delete `.github/CODEOWNERS` from your branch or replace it with setting specific to your experiment. Example: [#26](https://github.com/dotnet/runtimelab/pull/26/files)
+   - Make sure to edit your experiment branch to just build the packages that need to be built from that branch. [Example PR](https://github.com/dotnet/runtimelab/pull/467). To read more about why this can cause issues, read [this issue](https://github.com/dotnet/runtimelab/issues/465).
 - If your experiment is branched from [dotnet/runtimelab:standalone-template](https://github.com/dotnet/runtimelab/tree/standalone-template) follow the [README.md](https://github.com/dotnet/runtimelab/tree/standalone-template#standalone-experiments).
+- To make sure we follow our naming conventions, make sure all packages produced on your experiment are prefixed with Microsoft.* or System.*
```

Comment:
I would quite both "Microsoft.*" and "System.*" and then add the missing period.

Reply:
Sounds good. I can add a back tick so that is shows nice in the markdown.